server/server_test.go

```diff
@@ -629,6 +638,14 @@ func TestCustomClientAuthentication(t *testing.T) {
 	if _, err := nats.Connect(addr, nats.UserInfo("invalid", "")); err == nil {
 		t.Fatal("Expected client to fail to connect")
 	}
+
+	s.opts.AlwaysEnableNonce = true
```

Comment:
I am getting races when running your test, so need to revisit that.

Reply:
OK, I was less lazy now, think it probably fixed now